tests/IntegrationTestCase.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace my127\Workspace\Tests;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Process\Process;
+use my127\Workspace\Tests\Util\Workspace;
+
+class IntegrationTestCase extends TestCase
+{
+    protected function setUp(): void
+    {
+        $this->workspace()->reset();
```

Comment:
The workspace will be reset (all contents deleted, folder created) before each test.